saleor_app_framework/app_framework/endpoints/manifest.py

```diff
@@ -0,0 +1,27 @@
+from fastapi import APIRouter, HTTPException
+from fastapi.requests import Request
+
+from ..core.manifest import get_app_manifest
+from ..schemas.manifest import Manifest
+
+router = APIRouter()
+
+
+@router.get(
+    "/manifest",
+    response_model=Manifest,
+    # FIXME validate manifest in runtime
+    responses={502: {"description": "Manifest not found."}},
+)
+async def manifest(
+    request: Request,
+):
```

Comment:
Not sure about this formatting, I think it should be:
```suggestion
async def manifest(request: Request):
```
?